### DISCUSSION TITLE
Create design files

### DISCUSSION THREAD

**Discussion by 6543** [woodpecker-ci/woodpecker#2549] [General]
answer to https://github.com/woodpecker-ci/woodpecker/issues/1109#issuecomment-1221387739 ;)

also important to  #468

**Comment by 6543** [comment]
https://github.com/woodpecker-ci/woodpecker/issues/468#issuecomment-1207138477 might give some hints

**Comment by qwerty287** [comment]
Can be closed in favor of https://github.com/woodpecker-ci/design, we can extend
